requirements-ts.txt

```diff
@@ -1,3 +1 @@
-sktime==0.5.3
-tbats==1.1.0
-pmdarima>=1.8.0
+sktime[all_extras]==0.5.3
```

Comment:
doesn't seem to be working fine on Windows (check CI)

Reply:
There will be too many optional installations here. Shall we use only what is needed?

Reply:
@Yard1 not sure about using fbprophet in Windows without conda might work, maybe we can skip `fbprophet` testing if platform is windows. More on the topic on this SO [question](https://stackoverflow.com/questions/60388880/import-pystan-api-failedimporterror-dll-load-failed-the-specified-module-cou).

Reply:
@ngupta23 I agree